cruise.umple/src/StateMachine_Code.ump

```diff
@@ -553,7 +553,7 @@ class State
 
   public String getType()
   {
-    return numberOfTransitions() == 0 && numberOfNestedStateMachines() == 0 ? "Simple" : "Complex";
+    return numberOfTransitions() == 0 && numberOfNestedStateMachines() == 0 && numberOfActions() == 0 ? "Simple" : "Complex";
```

Comment:
@vahdat-ab please see this [gist](https://gist.github.com/jblang94/ab5f29833b4b45de185c17fbec60f346) . It's easier to explain using one of the examples that caused me to add that check in.